tests/dragonfly/seeder/script-genlib.lua

```diff
@@ -192,6 +192,30 @@ function LG_funcs.mod_json(key, dbsize)
     end
 end
 
+-- streams
+-- store sequences of timestamped events
+
+function LG_funcs.add_stream(key, keys)
+    local entries = {}
+    local limit = LG_funcs.csize
+    for i = 1, limit do
+        table.insert(entries, tostring(i))
+        table.insert(entries, randstr())
```

Comment:
I believe you should use the randstr_sequence functions.
See how it is used in LG_funcs.add_hash